querygen/ujquery.go

```diff
@@ -37,6 +37,7 @@ func NewUserJourneyQuery(index string) *UserJourneyQuery {
 			"event_verb",
 			"kind",
 			"messageId",
+			"namespace",
```

Comment:
not sure I'm following - the `namespace` field was removed from multiple functions, why is it here? should'nt it include workspaceID?

Reply:
There is no workspaceID field in Splunk, so we query the namespace and convert it to workspaceID via jq.

I removed it elsewhere because here it adds it to all queries